selenium_tests/articles/test_comments.py

```diff
@@ -0,0 +1,130 @@
+from selenium.webdriver.common.by import By
+from selenium_tests.base import BaseSeleniumTests
+from home.factories import (
+    SectionFactory,
+    ArticleFactory,   
+)
+from comments.models import CommentStatus
+from iogt_users.factories import (
+    AdminUserFactory,
+    UserFactory
+)
+from selenium_tests.pages import ArticlePage
+
+class ArticleCommentsSeleniumTests(BaseSeleniumTests):
+
+    def setUp(self):
+        self.setup_blank_site()
+        self.user = AdminUserFactory()
+        self.user2 = UserFactory()
+        self.user3 = UserFactory()
+        self.section = SectionFactory(parent=self.site.root_page)
+        self.article01 = ArticleFactory(parent=self.section, title = 'article01')
+        self.article02 = ArticleFactory(parent=self.section, title = 'article02', commenting_status = CommentStatus.CLOSED)
+
+    def test_basic_article_comment(self):
+        #login as an admin so we can leave a comment
+        login_page = self.visit_login_page()
+        login_page.login_user(self.user)
+        
+        #visit an article and leave a comment
+        test_comment = 'test comment here'
+        self.visit_page(self.article01)
+        article_page = ArticlePage(self.selenium)
+        article_page.comments_section.submit_comment(test_comment)
+        self.visit_page(self.article01)
+        self.assertIn(test_comment, article_page.comments_section.retrieve_comments())
+
+    def test_remove_article_comment(self):
+        #login as an admin
+        login_page = self.visit_login_page()
+        login_page.login_user(self.user)
+        
+        #visit an article and leave a comment
+        test_comment = 'test comment here'
+        self.visit_page(self.article01)
+        article_page = ArticlePage(self.selenium)
+        article_page.comments_section.submit_comment(test_comment)
+        self.visit_page(self.article01)
+
+        #delete comment
+        article_page.comments_section.delete_last_comment()
+        body_text = self.selenium.find_element(By.TAG_NAME, 'body').text
```

Comment:
Slight variations of this and the following line of code appear again and again. Is there any way to consolidate this in a method or function?

Is it possible to be more precise than checking the entire body for a message string? This could make the test flakey in the future.